app/controllers/ask_trn_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+class AskTrnController < ApplicationController
+  include EnforceQuestionOrder
+
+  layout "two_thirds"
+
+  def new
+    @ask_trn_form = AskTrnForm.new(trn_request:)
+    @ask_trn_form.do_you_know_your_trn = session[:user_knows_trn]
+  end
+
+  def create
+    @ask_trn_form = AskTrnForm.new(trn_request:)
+    if @ask_trn_form.update(trn_params)
+      session[:user_knows_trn] = trn_params.require(:do_you_know_your_trn)
```

Comment:
Could we remove the need for this session variable? They're undesirable as they act like global variables, and make testing harder.

In this case, could we set the `trn_from_user` database field to `""` when the users specify they do not know their TRN, and then use that value at later points?